src/main/java/validation/BusinessSampleUnit.java

```diff
@@ -104,10 +104,6 @@
   @Pattern(regexp = NON_BLANK_INTEGER_RE)
   String cell_no;
 
-  @NotNull
-  @Size(min = 1, max = 4)
-  String formtype2;
```

Comment:
You still need a formtype in this DTO (this is the reason you are getting a null in party service).

Need to align the name for 'formtype' here: https://github.com/ONSdigital/rm-sample-service/blob/75cc7c3b4dab3971d6cf32d94ae9221fe125191d/src/main/java/uk/gov/ons/ctp/response/sample/ingest/CsvIngesterBusiness.java#L35
and here: https://github.com/ONSdigital/ras-party/blob/d919d5f5e91c29f1c73b09fcdf249cc1d162b7cf/ras_party/schemas/party_schema.json#L20

Reply:
Nah - formType is in the base class, I've only removed formType2.  The reason I was getting null in the party service was because I has erroneously removed PartyUtil.java:54 (which is where formtype gets set in the request to party).